package.json

```diff
@@ -165,11 +165,13 @@
     "test": "npm-run-all --parallel test:*",
     "test:js": "jest --config=tests/js/jest.config.js",
     "test:js:coverage": "npm run test:js -- --collectCoverage && open build/logs/lcov-report/index.html",
+    "test:js:debug": "node --inspect-brk node_modules/.bin/jest --config=tests/js/jest.config.js",
     "test:js:help": "npm run test:js -- --help",
     "test:js:watch": "npm run test:js -- --watch",
     "test:php": "vendor/bin/phpunit",
     "test:php:help": "npm run test:php -- --help",
     "test:e2e": "WP_BASE_URL=http://localhost:8899 jest --runInBand --config=tests/e2e/jest.config.js",
+    "test:e2e:debug": "PUPPETEER_DEVTOOLS=true WP_BASE_URL=http://localhost:8899 node --inspect-brk node_modules/.bin/jest --runInBand --config=tests/e2e/jest.config.js",
```

Comment:
I tried to document this here:

https://github.com/google/web-stories-wp/blob/chore/dev-docs/docs/e2e-tests.md#debugging

Basically, the idea is to allow debugging tests on the node context (e.g. debug your test), while dev tools would allow you to debug the actual editor code in the console. `debugger` would only work on the node context. To debug the editor you'd use dev tools.

Reply:
I don't think it's possible to use `debugger` for the editor code and make it work in dev tools. But haven't looked into it much.